### PR TITLE
Feature Flags

### PR DESCRIPTION
**Pull request by flipactual**
**What**: 

Introduces `@emdaer/shared` local package and EmdaerFeatureFlags

**Why**: 

So we can develop new features in the release branch without breaking existing functionality

**How**: 

A new root level local package (`@emdaer/shared`) was added that can be used in any of our packages via `require('@emdaer/shared/<sharedLibName>')`

**Checklist**:

- [x] Documentation
- [ ] Tests (N/A)
- [x] Ready to be merged 
- [x] Added myself to AUTHORS

### CODE REVIEW

**Comment by flipactual** [comment]
I would like to have this so that feature flags can be overridden via command line, but I think that can be done separately when needed

File: packages/cli/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@emdaer/core": "^1.2.2",
+    "@emdaer/shared": "file:../../shared",
```
Comment (infiniteluke) [inline packages/cli/package.json]: If someone installs @emdaer/cli will this work, or will this only work when developing emdaer locally?

Comment (flipactual) [reply packages/cli/package.json]: Good point – I'm not sure how best to accomplish this

Comment (infiniteluke) [reply packages/cli/package.json]: Pulling feature flags into core would fix this. If someone is truly shared, core seems like a good place

**Review by infiniteluke — APPROVED** [review]

**Pull request merged**